### DISCUSSION TITLE
GitHub Support: Profile [Report Abuse] form is broken

### DISCUSSION THREAD

**Discussion by meowdhavan** [community/community#111269] [Other Feature Feedback, Questions, & Ideas]
### GitHub Account Report

Hi! When I try to report an account (`Account > Block or Report > Report Abuse`), the `Send request` button does not work. Nothing happens when I click the button.

Developer Console shows the following error when clicking the button: `An invalid form control with name='contact[comments]' is not focusable.`

Thanks!

**Comment by ghostinhershell** [comment]
Hi @madhavan-raja, 

Thanks so much for reaching out. Can you try opening up the same page in another browser or [in an incognito window](https://www.computerworld.com/article/3356840/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) and see if you're able to submit the Report Abuse form then?

**Reply by meowdhavan** [reply]
Thanks for the reply! I tried but it did not work and I got the same error.
